cmd/task/task.go

```diff
@@ -148,8 +150,8 @@ func main() {
 		return
 	}
 
-	if list {
-		e.PrintTasksHelp()
+	if list || listAll {
+		e.PrintTasksHelp(listAll)
 		return
 	}
```

Comment:
Boolean arguments are usually discouraged for readability reasons. I'd consider having a separate function for each one:

```suggestion
	if list {
		e.ListTasksWithDesc()
		return
	}
	if listAll {
		e.ListAllTasks()
		return
	}
```

In the package you could then abstract them in s private function as below:

```go
func ListTasksWithDesc() {
 	listTasks(false)
}

func ListAllTasks() {
 	listTasks(true)
}

func listTasks(all bool) {
 	// ...
}
```

Reply:
resolved in https://github.com/therealkevinard/task/commit/42702e81b35c5b4960c94ff65577f6c4314b0ccd